src/properties/practicebot/gamepad.properties

```diff
@@ -1,3 +1,8 @@
 gamepad.port = 2
+
 button.catapult.extend = 1
 button.catapult.retract = 2
+button.catapult.intake.in = 3
+button.catapult.intake.out = 4
+
+button.action.test = 3
```

Comment:
Let's put `button.action.test` on a button that's less common to press. I would suggest 10.

Basically, I just don't want to forget that we have testing on a common button. I don't think there's anything to prevent multiple commands from being mapped to the same button by accident.